nix/third-party/gRPC-haskell/core/chs/Network/GRPC/Unsafe.chi

```diff
@@ -0,0 +1,2 @@
+C->Haskell Interface Version C->Haskell Compiler, version 0.28.6 Switcheroo, 25 November 2017
```

Comment:
Why are we checking in the .chi files? Aren't then unnecessary is we check in the .hs?

Reply:
I removed them.